### DISCUSSION TITLE
【灌水】莫妮卡大姐姐的新单曲你们听了吗

### DISCUSSION THREAD

**Discussion by nana26710** [share121/inter-knot#4700] [灌水]
莫妮卡大姐姐的新单曲MV你们看了吗?
感觉好看里有一丝难听，难听里又透露着好看。
眼睛很享受，耳朵很痛苦。
（搬运）
![IMG_20240909_201242](https://github.com/user-attachments/assets/75c24821-1b23-47be-a394-bd7ef3150c71)

**Comment by BuYaoHoYo** [comment]
我个人感觉，莫妮卡大姐姐只适合当演员

**Reply by nana26710** [reply]
> 我个人感觉，莫妮卡大姐姐只适合当演员

她说这是回馈粉丝…现在看来，明明就是虐待粉丝嘛

**Reply by BuYaoHoYo** [reply]
就是
